doc/user/content/release-notes.md

```diff
@@ -103,6 +103,11 @@ List new features before bug fixes.
 
 {{% version-header v0.10.1 %}}
 
+- Correctly autogenerate views from Postgres sources during [`CREATE
+  VIEWS`](/sql/create-source/postgres/#creating-views) when the upstream table
+  contains numeric columns with no specified scale and precision. {{% gh 9268
+  %}}
+
```

Comment:
What happens if we ingest a numeric value that doesn't fit in our numeric type?

Reply:
The source will fail with an error message like this one:

```
materialize=> SELECT * FROM foo;
ERROR:  "1234567890123456789012345678901234567890" is out of range for type numeric: exceeds maximum precision 39
```